pkg/daemon/server/service/pipeline_metrics_query.go

```diff
@@ -55,9 +55,9 @@ type PipelineMetadataQuery struct {
 }
 
 const (
-	PipelineStatusOK       = "OK"
+	PipelineStatusHealthy  = "healthy"
```

Comment:
What is the reason of this sort of inconsistency (upper case, lower case)?

Reply:
I formatted to match constants as defined in the UI code, uppercase for pipeline running status and lowercase for pipeline health

Reply:
Can we rename the constant names? There's no way to tell at all.